create_jenkins_job.py

```diff
@@ -162,11 +162,13 @@ def main(argv=None):
         # many, likely related to qemu). Also disable linter tests (because
         # this is already a very slow job and because we lint plenty on other
         # jobs). Also don't build packages in parallel because we've seen hung
-        # builds that seem to be caused by parallelism.
+        # builds that seem to be caused by parallelism. Also don't rerun tests,
+        # as we're currently not paying attention to test failures.
         if os_name == 'linux-aarch64':
             job_data['dont_notify_every_unstable_build'] = 'true'
             job_data['ament_test_args_default'] = '--retest-until-pass 10 --ctest-args -LE linter --'
             job_data['ament_build_args_default'] = ''
+            job_data['ament_test_args_default'] = ''
```

Comment:
I went back to running the tests but excluding the linter tests (check the latest commits on this branch). I suppose that we could just disable tests, but I feel that running them once is a good middle ground between no testing and full testing, so I'd rather keep doing that if we can afford the CPU time for it.